packages/indexer/src/EventHandlers.ts

```diff
@@ -16,6 +16,10 @@ import { TryFetchIpfsFile } from "./utils/ipfs"
 import { GetStatusId, GetVisibilityId, Slugify } from "./utils/mapping"
 import { decodeAbiParameters } from 'viem'
 import dayjs from 'dayjs'
+import { ConditionModule } from "./src/DbFunctions.bs"
```

Comment:
Ah, sorry I think this is an unused import. I was using it for the type, but not anymore...